zoo/dmc2gym/envs/dmc2gym_lightzero_env.py

```diff
@@ -330,13 +373,13 @@ def reset(self) -> Dict[str, np.ndarray]:
             obs = obs['state']
 
         obs = to_ndarray(obs).astype(np.float32)
-        action_mask = None
 
         self._timestep = 0
         if self._save_replay_gif:
             self._frames = []
-
-        obs = {'observation': obs, 'action_mask': action_mask, 'to_play': -1, 'timestep': self._timestep}
+            
+        action_mask = -1
```

Comment:
it would be better to set the default value to 1, since 1 means no mask

Reply:
To be consistent with other variables (like to_play), -1 means a special flag.